.sync/devcontainer/devcontainer.json

```diff
@@ -0,0 +1,12 @@
+{
+  "image": "ghcr.io/tianocore/containers/fedora-35-dev:latest",

```

Comment:
If we want to keep the devcontainer on the latest image (which I think is fine), we would add another entry to `.sync/Version.njk` like `linux_gcc_container_dev` and set it to this then that is centralized as well.

Reply:
I think this one if fine to leave locally defined since its intended for local development and is the only place where it will be used so far so I think I may leave it for a sake of readability. We wont be changing it much anyways since the its using latest, which I feel just avoids churn for a less critical versioning.